corgidrp/detector.py

```diff
@@ -726,18 +701,23 @@ def create_onsky_flatfield(dataset, planet=None,band=None,up_radius=55,im_size=4
         yc = int(centroid[1])
         up_radius=up_radius
         smooth_images.append(planet_image)
+        planet_image_err=np.random.normal(np.nanmedian(planet_image),5,(image_size[0],image_size[1]))
```

Comment:
Can you please explain this line, @ramya-anche ? I'm not sure I'm following, sorry.